boards/nxp/frdm_ke17z512/doc/index.rst

```diff
@@ -0,0 +1,195 @@
++-----------+------------+-------------------------------------+
+| PINCTRL   | on-chip    | pinctrl                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | uart                                |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+``boards/nxp/frdm_ke17z512/frdm_ke17z512_defconfig``.
+
+Other hardware features are not currently supported by the port.
+
+System Clock
+============
+
+The KE17Z9 SoC is configured to run at 48 MHz using the FIRC.
+
+Serial Port
+===========
+
+The KE17Z9 SoC has three LPUARTs. UART2 is configured for the console.
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the :ref:`opensda-jlink-onboard-debug-probe`.
+
+Using LinkServer: :ref:`opensda-daplink-onboard-debug-probe`
+------------------------------------------------------------
+
+Install the :ref:`linkserver-debug-host-tools` and make sure they are in your
+search path.  LinkServer works with the CMSIS-DAP firmware included in the
+on-board debugger.
+
+Use the ``-r linkserver`` option with west to use the linkserver runner.
+
+.. code-block:: console
+
+   west flash -r linkserver
+
+Using JLink
+-----------
+
+Install the :ref:`jlink-debug-host-tools` and make sure they are in your search
+path.
+
+Follow the instructions in :ref:`opensda-jlink-onboard-debug-probe` to program
+the `Segger J-Link OpenSDA V2.1 Firmware`_. Note that Segger
+does provide an OpenSDA J-Link Board-Specific Firmware for this board, however
+it is not compatible with the DAPLink bootloader.
+
+JLink is the default for this board, ``west flash`` and ``west debug`` will
+call the jlink runner.
+
+Configuring a Console
+=====================
+
+Regardless of your choice in debug probe, we will use the OpenSDA
+microcontroller as a usb-to-serial adapter for the serial console.
+
+Connect a USB cable from your PC to J10.
+
+Use the following settings with your serial terminal of choice (minicom, putty,
+etc.):
+
+- Speed: 115200
+- Data: 8 bits
+- Parity: None
+- Stop bits: 1
+
+Flashing
+========
+
+Here is an example for the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: frdm_ke17z512
+   :goals: flash
+
+Open a serial terminal, reset the board (press the SW1 button), and you should
+see the following message in the terminal:
+
+.. code-block:: console
+
+   *** Booting Zephyr OS build xxxxxxxxxxxx ***
```

Comment:
Is this the correct output? If not please update.

Reply:
Yes, this is the correct output

Reply:
I don't think it says `xxxxxxxxxxx`

Reply:
@ankeXiao what the others are saying is that the xxxxx field is supposed to show what version of zephyr you built with. From a documentation perspective, please use what you see now.